src-tauri/src/mm_proxy_adapter.rs

```diff
@@ -156,14 +155,13 @@ impl ProcessAdapter for MergeMiningProxyAdapter {
             args.push(format!("merge_mining_proxy.monerod_url={}", node));
         }
 
-        // TODO: uncomment if p2pool is needed in CPU mining
         if config.p2pool_enabled {
             args.push("-p".to_string());
             args.push("merge_mining_proxy.p2pool_enabled=true".to_string());
             args.push("-p".to_string());
             args.push(format!(
                 "merge_mining_proxy.p2pool_node_grpc_address={}",
-                "https://grpc-p2pool.nextnet.tari.com:443"
+                format!("http://127.0.0.1:{}", config.p2pool_grpc_port)
```

Comment:
So won't this mean we _never_ use the remote p2pool node?

Reply:
good point